### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by RakshithBhat03**
# Initial setup for pomodoro

## Features
- Landing Page
- Tasks Page
- Theme toggle
- Router setup

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
-    transform: rotate(360deg);
-  }
-}
+.wrapper {
+  min-height: 100vh;
+  min-width: 100vw;
+}
+.theme-default {
+  background-color: rgb(217, 85, 80);
+  color: rgb(217, 85, 80);
```
Comment (MonikaJhingan) [inline src/App.css]: Use of var is preferrable.

File: src/components/nav/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { useTheme } from "../../context";
+import { DARK, DEFAULT } from "../../constants";
+import { setLocalStorage } from "../../utils/localStorage";
+
+function Nav() {
+  const { theme, setTheme } = useTheme();
+  useEffect(() => {
+    if (theme === DARK || theme === DEFAULT) {
+      setLocalStorage("theme", theme);
+    }
```
Comment (MonikaJhingan) [inline src/components/nav/Nav.jsx:13]: Theme setting is done well.

File: src/pages/Tasks/Tasks.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.tasks-wrapper {
+  max-width: 50rem;
+  height: calc(100vh - 50px);
+  opacity: 0.9;
+  opacity: 0.9;
+}
+.tasks-container {
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.1);
```
Comment (MonikaJhingan) [inline src/pages/Tasks/Tasks.css]: Pls use var for colors.

**Review by MonikaJhingan — APPROVED** [review]
The UI looks good and the folder structure is also nice. Indentation and implementation is done carefully.

File: src/pages/Tasks/Tasks.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { AddFirstTask } from "../../components";
+import "./Tasks.css";
+import { useDocumentTitle } from "../../hooks";
+import "./Tasks.css";
```
Comment (abhij1607) [inline src/pages/Tasks/Tasks.jsx]: line 5 is repeating

**Review by abhij1607 — APPROVED** [review]
good folder structure
Neat and clean code

Can be merged

**Pull request merged**